src/SFA.DAS.AssessorService.Web/Controllers/OrganisationSearchController.cs

```diff
@@ -112,6 +112,16 @@ public async Task<IActionResult> Confirm(OrganisationSearchViewModel viewModel)
 
             if (organisationSearchResult != null)
             {
+                if (organisationSearchResult.CompanyNumber != null)
+                {
+                    var isActivelyTrading = await _organisationsApplyApiClient.IsCompanyActivelyTrading(organisationSearchResult.CompanyNumber);
+
+                    if (!isActivelyTrading)
+                    {
+                        return View("~/Views/OrganisationSearch/CompanyNotActive.cshtml", viewModel);
```

Comment:
Shouldn't _really_ be returning a View from a POST (should be returning a redirect to a GET).  But if the rest of this method is doing the same... 🤷‍♂